src/site/site.xml

```diff
@@ -79,15 +79,16 @@
     </head>
     <menu name="Apache HBase Project">
       <item name="Overview" href="index.html"/>
-      <item name="License" href="licenses.html"/>
+      <item name="License" href="https://www.apache.org/licenses/"/>
```

Comment:
Do we need this in the header? The previous PR added a link to More Info section for index.html. Maybe the <item> could be removed completely.

Reply:
After checking a few Apache project websites I noticed that most of them have a link to ASF Licenses in the menu bar and it is also included in the footer. Based on this I'm fine with this link in the menu.